Changes

```diff
@@ -1,3 +1,13 @@
+10.4.3.2 (relative to 10.4.3.1)
```

Comment:
Sorry, missed this in my original review - we leave as-yet-unpublished versions as `10.4.x.x` so that we know they're not released yet (and can decide whether it's a minor or patch bump based on all changes that accumulate before release).

Reply:
done